### DISCUSSION TITLE
UXF on Hololens (UWP)

### DISCUSSION THREAD

**Discussion by CharloCamaro** [immersivecognition/unity-experiment-framework#135] [Q&A]
Hello!

Just wondering if anyone had any luck running/adapting UXF on Hololens (UWP) for experiments. I am having some trouble figuring out how to save exp. data, as It looks like the basic datahandlers (e.g. FileSaver/POST)  are not compatible with "WSAPlayer" platforms.

Thank you!

**Comment by jackbrookes** [comment]
HI! Sorry this has not been tested, and probably won't be a priority for now. Thanks!
